simple-wp-helpdesk/src/Rest/ViewsController.php

```
@@ -0,0 +1,275 @@
+		);
+	}
+
+	/**
+	 * Check that the current user can access view endpoints.
+	 *
+	 * @param WP_REST_Request $request REST request.
+	 */
+	public function permissions_check( WP_REST_Request $request ): bool {
+		unset( $request );
+		return current_user_can( 'edit_posts' );
+	}
```

PHPMD (2.15.0)

[error] The method permissions_check is not named in camelCase. (undefined)

(CamelCaseMethodName)